devbin/dev_proxy.py

```diff
@@ -50,12 +50,15 @@ async def render_html(request: web.Request, context: dict):
     return await render_template(SERVICE, request, **context, cookie_domain='localhost:8000')
 
 
+BC = web.AppKey('backend_client', Session)
```

Comment:
can we move the variable definition above the `proxy` function, since that uses it too?

Reply:
yup!